src/component/LabelList.tsx

```diff
@@ -52,6 +52,7 @@ export function LabelList<T extends Data>({ valueAccessor = defaultAccessor, ...
         const value = isNullish(dataKey)
           ? valueAccessor(entry, index)
           : (getValueByDataKey(entry && entry.payload, dataKey) as string | number);
+        console.log({ entry, value });
```

Comment:
Need to remove this

Reply:
https://github.com/recharts/recharts/pull/5676